dailizabot/text_patterns.py

```diff
@@ -13,4 +13,20 @@
     "Würde {0} dir denn wirklich helfen?",
     "Bist du sicher, dass du {0} brauchst?"]],
 
+    [r"Du bist (.*)", 
+     ["Wieso denkst du das {0}?",
+      "Kann ich etas dagegen tun {0}?", 
+      "Dankeschön, dass ich {0} bin!"]]
+
+    [r"Kannst du (.*)?", 
+     ["Ich kann {0} gerne versuchen!"]
+     ["Das kann ich leider nicht machen!"]
```

Comment:
```suggestion
     "Das kann ich leider nicht machen!",
```
See above, removed brackets, added comma.